package.json

```diff
@@ -28,9 +28,6 @@
     "test-coverage": "jest --coverage",
     "semantic-release": "semantic-release"
   },
-  "dependencies": {
-    "fs-monkey": "^1.0.0"
```

Comment:
Don't forget to run `yarn install` & commit the changes to the `yarn.lock` :)

Reply:
done